x-pack/plugins/infra/public/components/lens/lens_wrapper.tsx

```diff
@@ -104,7 +102,6 @@ export const LensWrapper = ({
     <div
       css={css`
         position: relative;
-        border-radius: ${euiTheme.size.s};
```

Comment:
cosmetic:

now:
<img width="402" alt="image" src="https://github.com/elastic/kibana/assets/2767137/86e2362a-137a-40b6-bd13-5dd01fc8edbe">

then:
<img width="434" alt="image" src="https://github.com/elastic/kibana/assets/2767137/08c22c71-c670-4022-bc99-35db8fb7b804">

Reply:
Nice, thanks for the fix!